src/rust/bitbox02-rust-c/src/bitboxbase.rs

```diff
@@ -207,7 +207,7 @@ pub extern "C" fn bitboxbase_state_set_not_alive() {
 #[no_mangle]
 pub extern "C" fn bitboxbase_state_get() -> BitBoxBaseBackgroundState {
     let state = unsafe { &STATE };
-    state.state.clone()
+    state.state
```

Comment:
@NickeZ clippy warned this

https://rust-lang.github.io/rust-clippy/master/index.html#clone_on_copy

Sounds legit, do you remember why there was a clone() here?

Reply:
Nope, can't remember. Usually I add `clone()`s as temporary fixes to make the borrowchecker happy. Having many clones is a bit of a code smell.